components/MapPage.qml

```diff
@@ -30,6 +30,14 @@ PageWithBottomEdge {
 
         head.actions: [
             Action {
+                id: navigateAction
```

Comment:
Agreed. Having used it a little on my phone, "Where am I?" gets used way more often.